lib/blinkist-airbrake-scrubber/hash.rb

```diff
@@ -0,0 +1,22 @@
+# Add Hash#traverse(&block)
+class Hash
```

Comment:
Never ever extend the core classes unless you have a very good reason for it. There is no such reason here!

Reply:
Apart from the possibility to of method name collision, is there really a reason for not doing that? It's like the "poster feature" of Ruby: _you can easily extend any classes_ ;-) (also, I acknowledge your feedback about it and gonna try to find a better solution, too)

Reply:
The possibility of name collision was big enough of a problem that ruby language developers tried to solve this with an abomination of a feature called refinements: https://docs.ruby-lang.org/en/2.4.0/syntax/refinements_rdoc.html

> It's like the "poster feature" of Ruby: you can easily extend any classes ;-) 

yes, and it is a bad one from software design perspective. In practice it makes refactoring and debugging much harder.

don't get me wrong: it can be used productively and can be super useful, but you need to have a very good reason for doing so. "Because I can" does not cut it!

Reply:
Sure, will create a separate class like you suggested with `DeepTraversal`. I wanted a 'quick little fix' with that Hash#traverse method, and I'm not attached to this particular implementation :-)